build-cmakelibs.sh

```diff
@@ -126,10 +122,8 @@ build SDL_image -DCMAKE_POSITION_INDEPENDENT_CODE=OFF
 build SDL_ttf -DCMAKE_POSITION_INDEPENDENT_CODE=OFF -DSDL2TTF_SAMPLES=OFF
 
 build enet
+build argtable3 -DARGTABLE3_INSTALL_CMAKEDIR=$PS2SDK/ports\lib/cmake/ -DARGTABLE3_REPLACE_GETOPT=OFF -DARGTABLE3_ENABLE_EXAMPLES=OFF -DARGTABLE3_ENABLE_TESTS=OFF
```

Comment:
Might want to wrap `-DARGTABLE3_INSTALL_CMAKEDIR=$PS2SDK/ports\lib/cmake/` in quotes, and fix the backslash there